tests/test_module.py

```diff
@@ -15,6 +15,19 @@
 
 ALLOWED_ERROR = 0.001
 
+def assertInitArgsFail(self):
```

Comment:
@ashtul this should be a class method, espcially if you use `self`

Reply:
Im also not sure what are you trying to achieve here

Reply:
@ashtul  This checks for the initialization arguments. We didn't test for it before

Reply:
@danni-m This checks for the initialization arguments. We didn't test for it before

Reply:
@danni-m  This checks for the initialization arguments. We didn't test for it before

Reply:
> @ashtul this should be a class method, espcially if you use `self`

This means to duplicate the code for each test since it uses a different class.